src/components/layout/header/header.tsx

```diff
@@ -1,8 +1,19 @@
+import { ROUTE_PATH } from '@/constant/routes';
+import useLogoutMutate from '@/hooks/useLogoutMutate';
+import { removeLocalStorage } from '@/utils/storage';
 import { Box, Button, Toolbar, Typography } from '@mui/material';
 import AppBar from '@mui/material/AppBar';
+import { useNavigate } from 'react-router-dom';
 
 const Header = () => {
-  const handleLogout = () => {};
+  const navigate = useNavigate();
+  const logoutMutation = useLogoutMutate();
```

Comment:
요런 느낌으로 표현해주는게 더 괜찮지 않을까요?
```suggestion
  const { mutate:logoutMutate } = useLogoutMutate();
```